### DISCUSSION TITLE
No access to the MariaDB of Photoprism

### DISCUSSION THREAD

**Discussion by topa-LE** [photoprism/photoprism#4632] [Q&A]
Hello, I have not been able to access the mariaDB database for a week now. I have not changed anything in the Docker container.

Both containers are also up to date: mariaDB Vers. 11 and **Photoprism : latest**
Why is PP trying to **access port :53**? Something just doesn't fit here anymore. Unfortunately PP is currently no longer usable.

```
/opt/photoprism/bin/photoprism start
time="2024-11-18T09:54:43Z" level=info msg="config: waiting for the database to become available"
time="2024-11-18T09:55:43Z" level=error msg="dial tcp: lookup mariadb on 127.0.0.11:53: no such host"
```
Thanks

**Comment by dssysolyatin** [comment]
Photoprism tries to resolve the IP address of the mariadb container by using Docker’s internal DNS server. Docker’s DNS server lets you find a container’s IP address by using its name. If you’re not familiar with DNS resolving, you can read more about it [1]. So, the error means that the Photoprism container cannot find the IP address of the mariadb container.

If you haven’t changed anything in your Docker Compose file, try restarting the Docker service (dockerd). This might fix the issue, as there could be a bug. Before doing that, double-check that you haven’t made any changes to the Docker Compose file or updated firewall rules (like iptables or nftables).

If anything doesn’t help, please share the Docker Compose file you are using.

[1] https://www.cloudflare.com/en-gb/learning/dns/what-is-dns/
